scout/server/app.py

```diff
@@ -74,7 +74,7 @@ def create_app(config_file=None, config=None):
     def check_user():
         if not app.config.get("LOGIN_DISABLED") and request.endpoint:
             # check if the endpoint requires authentication
-            static_endpoint = "static" in request.endpoint or "report" in request.endpoint
+            static_endpoint = "static" in request.endpoint
```

Comment:
🙄

Reply:
I suppose this is a leftover from some version of Chanjo- or other reports integration. Did you by any chance try this on stage and click through a case with all available report types? 😊

Reply:
Yes, probably from Chanjo. I'll install the branch on stage, better to check properly, I agree!

Reply:
It's on stage now. Looks like all the reports redirect to the login page as expected.

Reply:
Thanks!